package.json

```diff
@@ -39,8 +39,6 @@
     "cover": "nyc --reporter=lcov mocha --require intelli-espower-loader build/test/*.js && nyc report",
     "docs": "jsdoc -c .jsdoc.js",
     "generate-scaffolding": "repo-tools generate all",
-    "lint": "eslint src/ system-test/ test/ conformance/",
```

Comment:
We kept these around in other repos because of samples.  Does this repo keep samples in it?

Reply:
It does, but they have their own package.json:

https://github.com/googleapis/nodejs-firestore/blob/master/samples/package.json

Should I add it back either way?